maven/jdk11/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM jenkins/inbound-agent:alpine as jnlp
 
-FROM maven:3.8-jdk-11-slim
+FROM maven:3.8.2-eclipse-temurin-11
```

Comment:
does this image define a jdk in the location you want?

Reply:
Not at all: these are the excpetion (the layers does not have the same order as the others still present).

Shall we install maven by ourselves instead? I mean, the value of using this official image is low: we only have to download a zip and extract it somewhere + update the PATH right?